content/rs/databases/active-active/aa-planning.md

```diff
@@ -0,0 +1,82 @@
+- [**database replication**]({{<relref "/rs/databases/configure/replication.md">}}): enabling replication doubles memory consumption
+- [**Active-Active replication**]({{<relref "/rs/databases/active-active/_index.md">}}): enabling Active-Active replication requires double the memory of regular replication, which can be up to four times (4x) the original data size.
+- [**database replication backlog**]({{<relref "/rs/databases/active-active/manage-aa#replication-backlog/">}}) for synchronization between shards. By default, this is set to 1% of the database size.
+- [**Active-Active replication backlog**]({{<relref "/rs/databases/active-active/manage-aa.md">}}) for synchronization between clusters. By default, this is set to 1% of the database size.
+
+It's also important to know Active-Active databases have a lower threshold for activating the eviction policy, because it requires propagation to all participating clusters. The eviction policy starts to evict keys when one of the Active-Active instances reaches 80% of its memory limit. 
+
+For more information on memory limits, see [Memory management with Redis Enterprise Software]({{<relref "/rs/clusters/optimize/node-memory.md">}}), [Memory and performance]({{< relref "/rs/concepts/memory-performance/">}}), or [Database memory limits]({{<relref "/rs/databases/configure/memory-limit.md">}}).
+
+## Networking
+
+Network requirements for Active-Active databases include:
+
+- A VPN between each network that hosts a cluster with an instance (if your database spans WAN).
+- At least two (but no more than five) participating clusters.
+- A network connection to [several ports](#network-ports) on each cluster from all nodes in all participating clusters.
+- A [network time service](#network-time-service) running on each node in all clusters.
+
+Networking between the clusters must be configured before creating an Active-Active database. The setup will fail if there is no connectivity between the clusters.
+
+### Network ports
+
+Every node must have access to the REST API ports of every other node as well as other ports for proxies, VPNs, and the admin console. See [Network port configurations]({{<relref "/rs/networking/port-configurations.md">}}) for more details. These ports should be allowed through firewalls that may be positioned between the clusters.
+
+### Network Time Service {#network-time-service}
+
+Active-Active databases require a time service like NTP or Chrony to make sure the clocks on all cluster nodes are synchronized.
+This is critical to avoid problems with internal cluster communications that can impact your data integrity.
+
+See [Synchronizing cluster node clocks]({{<relref "/rs/clusters/configure/sync-clocks.md">}}) for more information.
+
+## Redis modules {#redis-modules}
```

Comment:
WDTY about advocating for RedisJSON?
And say:
RedisJSON in Redis Enterprise Software Private Preview

Reply:
I'd need @lanceleonard to weigh in this. Do we want to add mentions to private preview features to the public docs?

Reply:
Responding privately.